lib/model/rwfolder.go

```diff
@@ -1151,7 +1151,6 @@ func (f *sendReceiveFolder) handleFile(file protocol.FileInfo, copyChan chan<- c
 		available:        reused,
 		availableUpdated: time.Now(),
 		ignorePerms:      f.ignorePermissions(file),
-		version:          curFile.Version,
```

Comment:
If we were to do this, you most also remove this field from the struct declaration so there's nothing else mistakenly using the now uninitialized field.